client/src/components/ManageAccount/ManageAccount.module.css

```diff
@@ -44,7 +44,7 @@
     width: var(--column-width--small);
 
     /* To ensure these 3 panels are always in the same column together */
-    max-height: calc(33% - var(--vertical-buffer));
+    max-height: calc(50% - var(--vertical-buffer));
```

Comment:
**Problem.** This code change makes the comment inaccurate. 

<details>
<summary>How is Comment Inaccurate?</summary>

The "3 panels" are "Licenses", "3rd Party Apps", and "Password" ([reference](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/62ca01fd-4444-4d1c-aeb8-6db5aac20590/)). With this change, the 3 panels can exceed 33% height, thus overflow one column.

The result is not easily seen, because the Portal may not show as many panels and can exist in this section ([design](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/62ca01fd-4444-4d1c-aeb8-6db5aac20590/)).

</details>

<details>
<summary>Proposed Solutions</summary>

- (a) update comment to be accurate
- (b) explain why comment is temporarily inaccurate

    <details>
    <summary>Examples</summary>

    - ```suggestion
          /* CAVEAT: your_explanation_here */
          max-height: calc(50% - var(--vertical-buffer));
        ```
    - ```suggestion
          /* CAVEAT: 50% instead of 30% allows different layout until FP-1329 */
          max-height: calc(50% - var(--vertical-buffer));
        ```

    </details>

</details>